config/application.rb

```diff
@@ -16,4 +16,8 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
   end
+ 
+    # 認証トークンをremoteフォームに埋め込む
+    config.action_view.embed_authenticity_token_in_remote_forms = true
```

Comment:
This is only used for Ajax requests. Since we are not using ajax, Please remove this line.